### DISCUSSION TITLE
可以设置同时运行两个用户的隧道吗？

### DISCUSSION THREAD

**Discussion by wucangeo** [natfrp/wiki#309] [使用问题] [answered]
目前采用 `/opt/sakurafrp/frpc_linux_arm -f <你的Token>:<隧道ID>[,隧道ID[,隧道ID...]]` 的方式运行，只能输入一个token，可以同时运行两个token下的多个隧道吗？

**Comment by LTHPKBTE** [comment, marked as answer]
两个一起开就行了。请使用类似screen的软件或使用脚本。

**Reply by wucangeo** [reply]
我又注册了一个Service，服务里面运行frpc，两个服务一起跑就可以了。
感谢回复！
